todo_app/templates/_card_partial.html

```diff
@@ -0,0 +1,47 @@
+<div class="col mb-2">
+    <div class="card text-white {{ "bg-danger" if item.overdue() else "bg-info" }} h-100">
+        <h5 class="card-header font-weight-bold">
+            {{ item.name }}
+        </h5>
+        <div class="card-body d-flex flex-column">
+            <div class="row">
+                {{ item.desc }}
+            </div>
+            <div class="row">
+                {{ "Due by: {0}".format(item.due) if item.due }}
```

Comment:
Really minor, but this could even just be a method on the Item class to format like `item.get_due_by_text()` or something (`item.due_by_text`, with `due_by_text` as a [property](https://www.programiz.com/python-programming/property)?)

Given the MVC pattern we're following, this could be quite nice in keeping the logic as simple as possible in our view, but I'm arguably taking it a little to the extreme!